generators/server/templates/src/main/java/package/web/rest/dto/_ManagedUserDTO.java

```diff
@@ -2,13 +2,20 @@
 <% if (databaseType == 'mongodb' || databaseType == 'sql') { %>
 import java.time.ZonedDateTime;<% } %>
 
+import java.util.Set;
+
 import <%=packageName%>.domain.User;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 
 /**
  * A DTO extending the UserDTO, which is meant to be used in the user management UI.
  */
 public class ManagedUserDTO extends UserDTO {
 
+    public static final int PASSWORD_MIN_LENGTH = 5;
```

Comment:
I retested the login with "user" and it seems to be working now, but to avoid backward compatibility issue maybe it would be best to leave de min password length to 4, like it was before in the UserDTO.